### ISSUE TITLE
Non USDM VIN support

### ISSUE THREAD

**Issue by ConnorRigby** [ConnorRigby/rx8-ecu-dump#2]
Apparently Japanese VINs are only 10 characters long, not 17 like in the US.

**Comment by ConnorRigby** [comment]
```
[RX8] [getVIN] payload length=24 strlen=20733                                                                           000000: 00 00 07 e8 49 02 01 53 45 33 50 31 31 36 31 36  ....I..SE3P11616                                               000010: 37 20 20 20 20 20 20 20 
```

Seems like the vin is padded out by space characters (0x20)

**Comment by ConnorRigby** [comment]
Fixed in [63726e3](https://github.com/ConnorRigby/rx8-ecu-dump/commit/63726e378d70c8f08a305648f3fd9f6d85c80325)
